### PR TITLE
[Theme save options] Save icons with themes + add small description

### PR DESCRIPTION
**Pull request by un1versal**
There's room for improvement on messages, this will hopefully make it clear what the option does and when should it be used.

By default option is should be **off** which is the recommended state.

**Before**
before

**After**
capture

### CODE REVIEW

**Comment by staticfox** [comment]
Is there a point to keeping the option at all for now? I tried this on a vanilla build and it still breaks. IMO I don't think there's a feasible way to get the option to work at all.

**Comment by wodim** [comment]
If there is "room for improvement in messages", I'd say that after this commit there is even more room for improvement.

**Comment by staticfox** [comment]
In my personal opinion, https://github.com/blindirc/KVIrc/commit/d0ca42300c18ec4f345a77981a7624e5041b6bfe would make a lot more sense since there is no chance of this working plus it gives us more time to focus on fixing the actual issue rather than dabble with long dialog messages and possible duplicate reports for people who smash next through option menus.

**Comment by un1versal** [comment]
> In my personal opinion, blindirc@d0ca423 would make a lot more sense since there is no chance of this working plus it gives us more time to focus on fixing the actual issue rather than dabble with long dialog messages and possible duplicate reports for people who smash next through option menus.

This PR has nothing to do with the bug report #1643, or because this option is broken.

And why do you think that #1645, #1314, and #1309 are related to #1643 imo its not as Ive experienced this icon corruption bug 3 times and none of those times was in conjunction or anywhere in the neighborhood of this, just hammering on theme changing while testing other issue can cause the icons to die.

Icon corruption bug is only the **big icons**. the small icons survive whatever  #1645, #1314, and #1309 cause is.

None the less. your PR makes sense temporarily So Ill support that with some minor changes to the notes.

After when #1643 is fixed  this one can go in as the reason why I requested this feature and why adding this message is due to reason explained on dialog messages and not #1643

To be honest the only reason why saving icons makes sense is if you want to replace them completely. But...

**Comment by pragmaware** [comment]
TL;DR :laughing: 
This text probably belongs to a "theme HOWTO" instead of the GUI. A shortened version could eventually end up in a tooltip for the option.

**Comment by un1versal** [comment]
This PR is already outdated, My local version is much shorter. but the issue is the text goes to middle of box. instead of just bellow the option.

**Comment by un1versal** [comment]
OK this is as ready as it will ever be.

capture
The text has been capped to minimum, the rest we can add to docs maybe when I figure how to do that and or tooltips

@pragmaware your button for a verdict.
